README.md

```diff
@@ -432,6 +432,47 @@ Note `[src]` above is an input to the component where aboves `iframe` 'lives'.
 
 :arrow_right: https://angular.io/guide/pipes
 
+### Vue.js
+
+Vue.js provides filters to allow for simple text formatting. The filter utilizes the `|` character which is appended to the expression followed by the filter's name.  
+
+Filters are usable within mustache interpolations.
+
+```html
+<h1>{{ name | lowercase }}</h1> 
+```
+
+Filters can also be used within the `v-bind` directive.
+
+```html
+<div v-bind:slug="slug | formatSlug"></div>
+```
+
+When creating filters, the function always receives the expression's value. 
+
+```js
+new Vue({
```

Comment:
I'm wondering if it might be simpler to just show an example of a global filter using `Vue.filter`. I think it would be a bit cleaner and (less boilerplate code) and it's more in line with how the examples declare components (also using the global registration).

Reply:
Do I understand correctly that `Vue.filter` will be available globally in app and this approach below only in a given component? Then it makes sense to give both examples because I think it's unique for Vue.js.

Reply:
Yes, that's correct. It works like that for components too so we might want to provide an example of that as well. In my mind though it's quite rare that one will want to create a component or filter only for a specific component so I figured it's not worth adding an example for those. But if the purpose is to show features that other frameworks don't have then I agree it makes sense to add it. In the interest of not repeating the same code just with a small difference and making the document more readable isn't it better to simply mention in a sentence that filters can also be declared globally / locally (depending on which one we chose to show here).

Reply:
Alright, that's true. Most of the time I want to keep the common features set comparison, unless something unique is popular in a given framework. If it's not then just a sentence mentioning it and a link should suffice.